### ISSUE TITLE
undefined is not a function

### ISSUE THREAD

**Issue by titarenko** [BohdanTkachenko/eslint-plugin-require-path-exists#13]
Hi!

It seems like ES6 feature is being used in your module's codebase. 

Could you please either specify on NPM which minimal version of node it does require or provide some solution for older versions, like, for example, employing [es6-shim](https://github.com/paulmillr/es6-shim)?

``` bash
node -v
v0.12.4
```

``` js
function resolveModule(value, fromDir, modulesDir) {
  var pathname = _url2.default.parse(value).pathname || '';

  if (pathname.startsWith('.')) {
```

``` bash
  1:17  error  Unexpected error in eslint-plugin-require-path-exists: undefined is not a function
TypeError: undefined is not a function
    at resolveModule (/usr/local/lib/node_modules/eslint-plugin-require-path-exists/lib/exists.js:70:16)
    at testModulePath (/usr/local/lib/node_modules/eslint-plugin-require-path-exists/lib/exists.js:209:11)
    at testRequirePath (/usr/local/lib/node_modules/eslint-plugin-require-path-exists/lib/exists.js:236:22)
    at EventEmitter.CallExpression (/usr/local/lib/node_modules/eslint-plugin-require-path-exists/lib/exists.js:282:7)
    at EventEmitter.emit (events.js:129:20)
    at NodeEventGenerator.enterNode (/usr/local/lib/node_modules/eslint/lib/util/node-event-generator.js:42:22)
    at CommentEventGenerator.enterNode (/usr/local/lib/node_modules/eslint/lib/util/comment-event-generator.js:98:23)
    at Controller.controller.traverse.enter (/usr/local/lib/node_modules/eslint/lib/eslint.js:767:36)
    at Controller.__execute (/usr/local/lib/node_modules/eslint/node_modules/estraverse/estraverse.js:397:31)
    at Controller.traverse (/usr/local/lib/node_modules/eslint/node_modules/estraverse/estraverse.js:495:28)  require-path-exists/exists
```

**Comment by BohdanTkachenko** [comment]
Hi @titarenko! This module is written in ES6, but before publishing to NPM, it is being compiled to ES5 with Babel to be compatible with old NodeJS versions. But the issue that you have probably appeared because there is no polyfill for String.prototype.startsWith (perhaps I forgot to enable Babel polyfill in code). I'll check if simply enabling polyfill will fix this.

**Comment by BohdanTkachenko** [comment]
This issue should be fixed in 1.1.5
Feel free to reopen this issue if it was not fixed for you
